### PR TITLE
Release 2017-04-04 11:48:38

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: sideci_python_sandbox/bad.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from __future__ import print_function
+import logging
```
Comment (pocke) [inline sideci_python_sandbox/bad.py:2]: __[staging][flake8]__
* [F401] 'logging' imported but unused

File: sideci_python_sandbox/bad.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+class Command(ScrapyCommand):
```
Comment (pocke) [inline sideci_python_sandbox/bad.py:3]: __[staging][flake8]__
* [F821] undefined name 'ScrapyCommand'

File: sideci_python_sandbox/eight_queen.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            if len(test_queens) == BOARD_SIZE:
+                return test_queens
+            else:
+                return add_queen(test_queens)
+        except BailOut:
+            pass
+    raise BailOut
+
+queens = add_queen([])
+print queens
```
Comment (pocke) [inline sideci_python_sandbox/eight_queen.py:27]: __[staging][flake8]__
* [E999] SyntaxError: invalid syntax

File: sideci_python_sandbox/bad.py (hunk 1 above)
Comment (sumyapp) [inline sideci_python_sandbox/bad.py:2]: __[flake8]__
* [F401] 'logging' imported but unused

File: sideci_python_sandbox/bad.py (hunk 2 above)
Comment (sumyapp) [inline sideci_python_sandbox/bad.py:3]: __[flake8]__
* [F821] undefined name 'ScrapyCommand'

File: sideci_python_sandbox/eight_queen.py (hunk 3 above)
Comment (sumyapp) [inline sideci_python_sandbox/eight_queen.py:27]: __[flake8]__
* [E999] SyntaxError: invalid syntax

**Pull request closed without merging**